resources/public/api/conf/1.0/application.yaml

```diff
@@ -413,24 +439,110 @@ components:
     ExciseMovement:
       type: object
       required:
-        - movementId
         - consignorId
         - localReferenceNumber
         - status
       properties:
-        movementId:
-          type: string
-          format: uuid
         consignorId:
           $ref: '#/components/schemas/ExciseReferenceNumber'
         localReferenceNumber:
           $ref: '#/components/schemas/LocalReferenceNumber'
-        administrativeReferenceCode:
-          $ref: '#/components/schemas/AdministrativeReferenceCode'
         status:
           $ref: '#/components/schemas/MovementStatus'
       xml:
         name: exciseMovement
+    ExciseMovement400Error:
```

Comment:
All of these error classes can be represented with one type, we would then just provide examples that override the fields for each example